srcalc/src/test/java/gov/va/med/srcalc/db/ResultsDaoIT.java

```diff
@@ -141,6 +150,7 @@ private void populateSampleResults()
         fResultsDao.persistSignedResult(fSampleResult2);
         fResultsDao.persistSignedResult(fSampleResult3);
         fResultsDao.persistSignedResult(fSampleResult4);
+        fResultsDao.persistHistoricalCalc(fHistoricalCalc5);
         simulateNewSession();
```

Comment:
I did it for the same reason as `makeReport()` above: encapsulating code in a method with a name enhances understandability.